We don't want a static binary, make sure the install target respects that.

diff -upr phpspy-0.5.0.orig/Makefile phpspy-0.5.0/Makefile
--- phpspy-0.5.0.orig/Makefile	2020-05-31 18:37:11.787907276 +0200
+++ phpspy-0.5.0/Makefile	2020-05-31 18:37:18.961241194 +0200
@@ -55,7 +55,7 @@ test: phpspy_static $(phpspy_tests)
 	printf "Passed %d out of %d tests\n" $$pass $$total ; \
 	[ $$pass -eq $$total ] || exit 1
 
-install: phpspy_static
+install: phpspy_dynamic
 	install -D -v -m 755 phpspy $(DESTDIR)$(prefix)/bin/phpspy
 
 clean:
